bsmschema/setup.cfg

```
@@ -12,11 +12,12 @@ classifiers =
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
     Programming Language :: Python :: 3.10
+    Programming Language :: Python :: 3.11
     Development Status :: 2 - Pre-Alpha
     License :: OSI Approved :: Apache Software License
 
 [options]
 python_requires = >=3.8
 install_requires =
-    pydantic
+    pydantic < 2.0a0
```

If they're going to fix it in the next release, we might as well do something that won't need to be undone:

```suggestion
    pydantic != 2.0a0
```

---

Well, I was not certain it will be fixed in the next release. Without `<` we will keep pulling alpha versions that are not ready for production (currently `2.0a1` but soon `2.0a2` or even` 2.0a3` perhaps without the fix).

---

Fixed by https://github.com/pydantic/pydantic/pull/5392/files after all, I guess the fix will be in **`2.0a2`**, so we only need skip **`2.0a1`**.

---

Changed to `pydantic != 2.0a1`.